site/_posts/2017-08-11-back-to-basics.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Last Week in ManageIQ: Back to Basics"
+author: jntullo
+date: 2017-08-15
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello there! It's [Jillian](https://github.com/jntullo) again, here with the latest happenings the last *two* weeks in ManageIQ. 
+I probably cannot top last week's [edition](http://manageiq.org/blog/2017/08/limericks-week-in-miq/), so I've decided to
+bring it back to basics for this one. Read on to see what's been happening in our world over the past two weeks. 
+
+## Featured
+
+If you're users of or contributors to the API, you may be interested to hear that the API now has its own repository!
+Head on over to its new location [here](https://github.com/ManageIQ/manageiq-api) and check out the [talk](http://talk.manageiq.org/t/proposed-split-repo-manageiq-api/2587/3) 
+article for more information.
+
+## Improved
+
+### Zone Selection
+Zone selection has been improved in the UI to include [all zones](https://github.com/ManageIQ/manageiq-ui-classic/pull/1769), not just the zones in the current region. 
```

Comment:
Based on the code change, it looks like it does the opposite.  Before it was showing `Zone.all` now it shows `Zone.in_my_region`